packages/plugin-user-agent-enrichment-browser/README.md

```diff
@@ -0,0 +1,75 @@
+<p align="center">
+  <a href="https://amplitude.com" target="_blank" align="center">
+    <img src="https://static.amplitude.com/lightning/46c85bfd91905de8047f1ee65c7c93d6fa9ee6ea/static/media/amplitude-logo-with-text.4fb9e463.svg" width="280">
+  </a>
+  <br />
+</p>
+
+# @amplitude/plugin-user-agent-enrichment-browser
+
+Official Browser SDK plugin for user agent enrichment.
+
+## Installation
+
+This package is published on NPM registry and is available to be installed using npm and yarn.
+
+```sh
+# npm
+npm install @amplitude/plugin-user-agent-enrichment-browser@^1.0.0
+
+# yarn
+yarn add @amplitude/plugin-user-agent-enrichment-browser@^1.0.0
+```
+
+## Usage
+
+This plugin works on top of the Amplitude Browser SDK. It's used for enriching your user agent information using the @amplitude/ua-parser-js npm package. The user agent identifies the application, operating system, vendor, and/or version of the requesting client. You can use this plugin to maintain the user agent information's consistency with earlier versions.
```

Comment:
```suggestion
This plugin works on top of the Amplitude Browser SDK. It's used for enriching events with user agent information using the @amplitude/ua-parser-js. The user agent identifies the application, operating system, vendor, and/or version of the requesting client.
```

Reply:
> You can use this plugin to maintain the user agent information's consistency with earlier versions.

can we replace this and be more specific why this plugin exists and why it could be useful for customers? It’s not clear versions were impacted and which “earlier versions” we want consistency with

Reply:
Thanks for pointing this out. Rephrase this sentence with some explanation.